admin/services/pupil.service.js

```diff
@@ -48,6 +48,22 @@ const pupilService = {
     }
   },
   /**
+   * Fetch one pupil filtered by pupil id and school id
+   * @param pupilId
+   * @param schoolId
+   * @returns {Promise.<*>}
+   */
+  fetchOnePupil: async (pupilId, schoolId) => {
```

Comment:
As the pupilId is a primary key, what is the purpose of the schoolId check?

Reply:
IIRC it was discussed that one pupil could be temporarily registered in more than one school (and not knowing yet how we are going to deal with this), the school id is to make sure we display the one for the school in the current session.